apps/api/src/routers/dev.py

```diff
@@ -0,0 +1,23 @@
+from fastapi import APIRouter
+from fastapi.responses import RedirectResponse
+
+from auth import user_identity
+from auth.user_identity import COOKIE_NAME, NativeUser
+
+router = APIRouter()
+
+
+@router.get("/impersonate/{ucinetid}")
+async def impersonate(ucinetid: str) -> RedirectResponse:
+    """Simulate a user identity during local development (does not require https)."""
+    user = NativeUser(
+        ucinetid=ucinetid,
+        display_name="Local Dev",
+        email="dev@irvinehacks.com",
+        affiliations=[],
+    )
+
+    res = RedirectResponse("/portal", status_code=303)
```

Comment:
Note: One thing for everyone to note is that when you fill out the application and click submit, nothing seems to happen. In the console, there is an error that is thrown about how the SendGrid API key is missing, but fortunately, the application data is still submitted to the local database, which can be verified by refreshing the page and seeing the portal instead of the application.

Reply:
This happens to me as well